src/hotspot/share/gc/shared/gc_globals.hpp

```diff
@@ -684,10 +684,10 @@
   develop(uintx, GCExpandToAllocateDelayMillis, 0,                          \
           "Delay between expansion and allocation (in milliseconds)")       \
                                                                             \
-  product(uintx, GCDrainStackTargetSize, 64,                                \
+  product(uint, GCDrainStackTargetSize, 64,                                \
           "Number of entries we will try to leave on the stack "            \
           "during parallel gc")                                             \
-          range(0, max_juint)                                               \
+          range(0, (max_juint-1)/2)                                               \
```

Comment:
```suggestion
          range(0, (UINT_MAX - 1) / 2)                                         \
```
Avoid using `juint_*` in code not interfacing java. Also the `\` is indented improperly, and even in above suggestion it might still be.